web/html/src/components/input/Form.tsx

```diff
@@ -137,12 +137,14 @@ export class Form extends React.Component<Props> {
             }
             return filtered;
           }, {});
-          const errors = Object.keys(this.props.errors || {}).reduce((filtered, key) => {
-            if (names.includes(key) && this.props.errors?.[key]) {
-              return filtered.concat(this.props.errors[key]);
-            }
-            return filtered;
-          }, []);
+          const errors = this.props.errors != null ?
```

Comment:
This ternary should be redundant with the latest updates, `Object.keys(this.props.errors || {}).reduce((filtered, key) => filtered, [])` will return `[]` in case that `this.props.errors` is null.

Reply:
To perhaps explain what I mean better, the current code:

```ts
const errors = this.props.errors != null ?
    Object.keys(this.props.errors || {}).reduce((filtered, key) => {
        if (names.includes(key) && this.props.errors?.[key]) {
            return filtered.concat(this.props.errors[key]);
        }
        return filtered;
    }, []) :
[];
```

Is equivalent to this code without the ternary:  

```ts
const errors = Object.keys(this.props.errors || {}).reduce((filtered, key) => {
    if (names.includes(key) && this.props.errors?.[key]) {
        return filtered.concat(this.props.errors[key]);
    }
    return filtered;
}, []);
```

Reply:
OK, just fixed it